### DISCUSSION TITLE
Display switch 180°

### DISCUSSION THREAD

**Discussion by ghost** [pixelit-project/PixelIt#377] [Help]
Hi, my clock is working fine, but i put it now on a place where the power Plug (who is on the right site) is not in place, i could make a new hole and it would fit. But i think maybe it is somehow possible to invert the text in 180 degree so i can just turn it 180. i cant find anything in the software. Is there a way to invert the text?

**Comment by foorschtbar** [comment]
You could try to add a `matrix->setRotation(180);` to the code and compile a custom version
